com.unity.mobile.notifications/Runtime/iOS/Plugins/UnityNotificationWrapper.m

```diff
@@ -99,7 +99,7 @@ void _ScheduleLocalNotification(struct iOSNotificationData* data)
     NSDictionary *userInfo = @{
         @"showInForeground": @(data->showInForeground),
         @"showInForegroundPresentationOptions": [NSNumber numberWithInteger: data->showInForegroundPresentationOptions],
-        @"data": @(data->data),
+        @"data": @(data->data ? data->data : ""),
```

Comment:
Does it have to be empty string? Can't be nil?

Reply:
According to Apple's document, it cannot be nil. https://developer.apple.com/documentation/foundation/nsdictionary, `Neither a key nor a value can be nil`.